lib/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+
+set(ofx_SRCS
+        messages.cpp
+        ofx_utilities.cpp
+        file_preproc.cpp
+        context.cpp
+        ofx_preproc.cpp
+        ofx_container_generic.cpp
+        ofx_container_main.cpp
+        ofx_container_security.cpp
+        ofx_container_statement.cpp
+        ofx_container_account.cpp
+        ofx_container_transaction.cpp
+        ofx_container_position.cpp
+        ofx_containers_misc.cpp
+        ofx_request.cpp
+        ofx_request_accountinfo.cpp
+        ofx_request_statement.cpp
+        ofx_sgml.cpp
+        ofc_sgml.cpp
+        win32.cpp
+        )
+
+set(ofx_HEADERS
+        messages.hh
+        ofx_preproc.hh
+        file_preproc.hh
+        context.hh
+        ofx_sgml.hh
+        ofc_sgml.hh
+        ofx_aggregate.hh
+        ofx_error_msg.hh
+        ofx_containers.hh
+        ofx_request.hh
+        ofx_request_accountinfo.hh
+        ofx_request_statement.hh
+        ofx_utilities.hh
+        tree.hh
+        win32.hh
+        )
+
+add_library(ofx ${ofx_SRCS} ${ofx_HEADERS})
+target_link_libraries(ofx OpenSP::OpenSP)
+target_include_directories(ofx PRIVATE ${ofx_HEADERS} ${COMMON_INCLUDE_DIRS})
```

Comment:
Change this line to remove `${ofx_HEADERS}` (the directive should only include directories, not files)